nbflow/extractor.py

```diff
@@ -57,7 +57,7 @@ def get_dependencies(self, dirnames):
                 sources = [self.resolve_path(filename, x) for x in params['__depends__']]
 
                 targets = params['__dest__']
-                if not hasattr(targets, '__iter__'):
+                if isinstance(targets, str):
```

Comment:
What is the motivation behind this change? `__dest__ = None` is a valid use of nbflow but `None` is not an instance of `str`, so this will not have the same behavior as before.